### ISSUE TITLE
Talent Choice Suggestion

### ISSUE THREAD

**Issue by 151m** [custom-ivan/atomic_war#10]
Some heroes have one very strong level 10 talent, and a very weak level 10 talent. For example Clinkz multishot at level 10, or Medusa split shot uses modifiers at 10, are both extremely strong, and the +health and +damage alternative talents are awful in comparison. Clinkz with the right talent vs with the wrong talent is like two different heroes, Medusa as well. This design choice is frustrating to play 50% of the time when I get the wrong talent.

I have a couple suggestions for alternatives to the current talent selection process:
-Have the same talents chosen automatically every game (for example, Clinkz always gets the split shot level 10 talent).
-Let players choose which talent they want each time a new talent is unlocked.
-Modify talents, so each choice is more balanced, this way getting the wrong talent doesn't hurt so much.

**Comment by Derceto** [comment]
Yeah I agree, I always wonder if I should be throwing back my level 10 dusa, clinkz, Crystal maiden, etc when I don't get the god tier lvl 10 talent, which isn't a particularly fun game mechanic. Seems like something comparable would be nice. Like for CM, 3 seconds of spell immunity when starting ult OR refund half cooldown of ult if ult is interrupted
